test_db.py

```diff
@@ -0,0 +1,30 @@
+import sqlite3
```

Comment:
Not sure why this file is here???

Reply:
whoops, just a little scipt to test the database is working, thought I had included it the `.gitignore`!